### DISCUSSION TITLE
Downloading 'marked for later' without deleting

### DISCUSSION THREAD

**Discussion by Selthorne** [nianeyna/ao3downloader#97] [Q&A]
So I was wondering if there was a way to download fics from the "marked for later" list without deleting them (I prefer to remove them as I read them). I've recently installed the latest version of the app as well as python 3.11.4 and I managed to download individual fics with the 'a' option just fine, but it doesn't seem to work for my marked for later or history. I also tried directly using 'm' but it's not working either so far. Everything seems to be okay aside from that (same menus/options as usual before downloads start). I also did the whole uninstall+reinstall and stopped at page 2 and not saving embedded images to avoid reaching the per-user click limit.

The app basically starts the logging-in process then stops after a few seconds, and nothing appears in the download folder.
The log file shows:
{"starting": "https://archiveofourown.org/users/[username]/readings?show=to-read", "timestamp": "07/11/2023, 00:17:22"}

![ao3downloader-screenshot-issue](https://github.com/nianeyna/ao3downloader/assets/139179747/4491d503-a3b1-4933-86bb-b6ec4a769aab)

**Comment by nianeyna** [comment]
This is, unfortunately, almost certainly due to the current DDoS attack on AO3 (as well as the measures they've taken to combat it). Normally you should absolutely be able to use the "a" option to do what you're trying to do - as far as I can tell you've hit on the correct approach. Unfortunately, it's just broken right now. :(

I'm definitely planning to work on a way around this problem - if possible with input from the ao3 dev team - but since the situation is volatile atm I'm currently mostly just waiting to see how things develop. Hopefully I'll have a better idea of what direction to take sometime in the next couple of days.

**Reply by Selthorne** [reply]
I thought it would be that. I tried during one of the "breaks" when the site was back but I didn't realise the new measures were already active (the false hope was painful).

Thank you again for everything you're doing with this app.
